requirements.txt

```diff
@@ -1,10 +1,10 @@
-kubernetes==9.0.1
-jupyterhub==1.1.0
-jupyterhub-kubespawner==0.12.0
+kubernetes==10.1.0
+jupyterhub==1.2.1
+git+https://github.com/jupyterhub/oauthenticator.git@15fa131853a047eb8acdfddfeb43da738812ed70
```

Comment:
Seems like new version of oauthenticator is out - 0.12.1 and it contains the change we need - can you update this to use it instead of git, please?